src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IArrayElementReferenceExpression.cs

```diff
@@ -518,19 +518,11 @@ public void F()
 }
 ";
             string expectedOperationTree = @"
-IInvalidOperation (OperationKind.Invalid, Type: ?, IsInvalid) (Syntax: '[0]')
-  Children(2):
-      ILiteralOperation (OperationKind.Literal, Type: System.Int32, Constant: 0, IsInvalid) (Syntax: '0')
-      IInvalidOperation (OperationKind.Invalid, Type: null, IsInvalid) (Syntax: '')
-        Children(0)
-";
-            var expectedDiagnostics = new DiagnosticDescription[] {
-                // CS1525: Invalid expression term '['
-                //         var a = /*<bind>*/[0]/*</bind>*/;
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "[").WithArguments("[").WithLocation(6, 27)
-            };
+IInvalidOperation (OperationKind.Invalid, Type: ?) (Syntax: '[0]')
```

Comment:
I'm trying to think of a way to preserve the intent of this test: how would `.?[0]` be parsed after this change?

Reply:
let me add a test for that.

Reply:
yeah, i can't figure a way to get an isolaed ElementAccessExpressionSyntax anymore.  THe issue being that you only get that when parsing the postfix of something.  And that thing will always succeed now as a collection expression since `[` is a legal start of one.